### ISSUE TITLE
Could not launch FontForge (exit code -1073741819)

### ISSUE THREAD

**Issue by sebi5361** [fontforge/fontforge#4471]
- I am on `Microsoft Windows 10 Pro 10.0.19042 Build 19042`
- To install FontForge I made sure to run the installer `FontForge-2020-03-14-Windows.exe` as Administrative User
- Same for the FontForge program itself `C:\Program Files (x86)\FontForgeBuilds\run_fontforge.exe` being run as Administrative User

...but I am getting an error at program startup:
<img src="https://user-images.githubusercontent.com/18078043/97774970-a7bc2a80-1b5c-11eb-964e-a33ea317b81f.png" height="200">
...and the program never shows up.

Note that `-1073741819 = 0xC0000005`

Please help!

## Check-list
- [x] Screenshot
- [x] The FontForge version and the operating system you're using
- [x] The behavior you expect to see, and the actual behavior
- [x] Steps to reproduce the behavior

**Comment by sebi5361** [comment]
I solved my issue!
I have some ideas of what could have caused this issue but I will not discuss it here as it requires some clarifications.
If someone falls on this post having the same issue, I will give some him/her kindly some feedback!
I am closing this thread.

**Comment by C-Windows-System32-cmd** [comment]
> I solved my issue!
> I have some ideas of what could have caused this issue but I will not discuss it here as it requires some clarifications.
> If someone falls on this post having the same issue, I will give some him/her kindly some feedback!
> I am closing this thread.

And why was the reason for the closure of this issue? What was the solution? I also have this problem on my Lenovo Yoga convertible running Windows 10 Pro 20H2

**Comment by ctrlcctrlv** [comment]
FontForge doesn't need to be run with administrative rights. Uninstall it, Reinstall and _when asked_ self-grant admin privileges, then run it as a normal user…

**Comment by AmhLupaga** [comment]
I have the exact same problem, but the thing is, it doesn't change anything whether I run it as administrative user or not. Basically I just download the installer, execute it, then launch FontForge and it doesn't work, saying the exact same thing with the same exit code. There so little I've done and so little online on this problem that I have absolutely no idea what I could have done wrong. The only difference from the case here is that I am on Windows 11, but if it was the issue, I think I would have found more people with the same problem. I'm at my wits end here.

EDIT: So I've tried a bunch of things more or less absurd, and one did kinda work. I actually have only one account on my computer, so I was wondering if maybe doing things from it would execute FontForge as administrative user by default (since the account is an administrative user). Well so I tried creating a dummy account and do things from it, turns out it works. I don't know why tho. I've tried experimenting, turning my main account to that of a simple user, and it still didn't work, so in the end this doesn't seem to be the cause. What's really absurd to me is that from both account I'm using the same application (I didn't uninstall+reinstall it each time), and for one of them it works while for the other it doesn't. So at least I'm confident it's not a problem with the installation of the app but rather that for some reason I can't access some files that are necessary to run the app from my main account.
Long story short, I can use FontForge, but only from another account than my main one, which is really tedious, so I'm still looking for some solution here.
